swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
```

Comment:
Nice use of `in` for the safety check. Is there a way we could use `remove` without doing this check? How does `remove` communicate that an item wasn't found? Can we handle that case?